tests/acceptance/features/bootstrap/OcisConfigContext.php

```diff
@@ -0,0 +1,121 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ * 
+ * @author Sajan Gurung <sajan@jankaritech.com>
+ * @copyright Copyright (c) 2023 Sajan Gurung sajan@jankaritech.com
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License,
+ * as published by the Free Software Foundation;
+ * either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+use Behat\Behat\Context\Context;
+use TestHelpers\OcisConfigHelper;
+use PHPUnit\Framework\Assert;
+
+ class OcisConfigContext implements Context {
+	/**
+	 * This is used to determine if the ocis config has been changed via tests
+	 * 
+	 * @var bool
+	 */
+	private static $touched = false;
```

Comment:
any specific reason to have this static?

Reply:
With this, we want to determine whether we have to restart the ocis server after each scenario (just to prevent unnecessary restarts)